pages/chat/[[...roomId]].js

```diff
@@ -136,16 +115,54 @@ export default function Chat() {
-                    .mx_RoomHeader { padding: calc(var(--margin) * 0.75) var(--margin); border-bottom: 1px solid var(--color-foreground-alpha); }
-                    
+                /* Don't display explore public rooms option */
+                .mx_HomePage_button_explore { display: none !important }
+                .mx_HomePage_default_buttons { display: initial !important }
+                /* Don't display element welcome message */
+                .mx_HomePage_default_wrapper > div:first-child { display: none }
+
+                .mx_SearchBar {
+                    position: absolute; right: 0; left: 0; bottom: 0; z-index: 10;
+                    background: var( --cpd-color-theme-bg);
+                    border-top: 1px solid var(--roomlist-separator-color);
+                    z-index: 999999999;
+
+                }
+
+                .mx_SearchWarning {
+                    background-color: var(--cpd-avatar-color);
+                    padding: 1rem;
+                    z-index: 9999999999;
+                }
+
+                .mx_RoomView_searchResultsPanel .mx_RoomView_messageListWrapper { padding-bottom: 80px; }
+                .mx_RoomView_messageListWrapper { padding-top: 140px; }
+
+                 @media (max-device-width: 1079px) {
+                    .mx_RoomHeader { padding: calc(var(--margin) * 0.75) var(--margin); border-bottom: 1px solid var(--color-foreground-alpha); 
+                    }
+
                     /* Make the "right panel" cover the full screen */
-                    .mx_RightPanel { position: fixed; left: 0; right: 0; bottom: 0; top: 0; z-index: 999999; }
+                    .mx_RightPanel { position: fixed; left: 0; right: 0; bottom: 0; top: 0; z-index: 999999; 
+                    }
+
+                @media ${breakpoints.phoneOnly} {
+                    .mx_RoomHeader { padding: 1rem var(--RoomView_MessageList-padding) }
 
-                    /* More breathing room in the main timeline of a chat */
-                    .mx_RoomView_timeline_rr_enabled .mx_EventTile[data-layout=group] .mx_EventTile_line,
-                    .mx_RoomView_timeline_rr_enabled .mx_EventTile[data-layout=group] .mx_ThreadSummary,
+                    .mx_RoomHeader_wrapper { flex-wrap: wrap }
+                    .mx_RoomHeader_avatar { flex: 0 1 1% }
+                    .mx_RoomHeader_name { font-weight: bold; flex: 1 0 }
+                    .mx_RoomTopic { flex: 0 0 100%; margin: 12px 6px }
                     .mx_RoomView_timeline_rr_enabled .mx_EventTile[data-layout=group] .mx_ThreadSummary_icon { margin-right: unset; }
+
+                    /* modal overlay when creating rooms */
+
+                    .mx_Dialog { width: 80% }
+                    .mx_CreateRoomDialog.mx_Dialog_fixedWidth { width: 100% }
+                    .mx_Dialog_fixedWidth { width: 100% }
+                    .mx_Dropdown_menu { max-width: 100% }
+
+                    #mx_JoinRuleDropdown__public { display: none }
```

Comment:
Let's add comments to each of these lines on what they intend to do.
This is especially helpful for later; we do not control if/how/when Element changes their markup. So we might need to make adjustments to all of these CSS rules whenever we update Element. Therefore it's important to remember everything these lines intend to do, so one can later double-check if they still do what they are supposed to do.

Reply:
Also, if the intention of this specific line is to disallow creating public rooms: Why is it inside a `@media ${breakpoints.phoneOnly} {`? Do we only want to disallow creating public rooms on smartphones? If so, why?

Reply:
Was the intention to take away this option from users for now?
<img width="482" alt="image" src="https://github.com/medienhaus/medienhaus-spaces/assets/706419/138d159f-b14c-4870-b805-e99b78162d27">